### PR TITLE
Code cleanup and add pickle5 in preparation of upgrade RAPIDS

### PR DESCRIPTION
**Pull request by rilango**

### CODE REVIEW

File: common/cuchemcommon/fingerprint.py [hunk 1]
```diff
@@ -10,16 +10,7 @@
...
-    import cupy # TODO is there a better way to check for RAPIDS?
-except:
-    logger.info('RAPIDS installation not found. Numpy and pandas will be used instead.')
-    import numpy as xpy
-    import pandas as xdf
-else:
-    logger.info('RAPIDS installation found. Using cupy and cudf where possible.')
-    import cupy as xpy
-    import cudf as xdf
+import cupy
```
Comment (mlgill) [inline common/cuchemcommon/fingerprint.py:13]: Just out of curiosity, was this removed because it's no longer part of the benchmarking framework?

**Review by mlgill — APPROVED** [review]
@rilango Approved, only one curiosity question.

File: common/cuchemcommon/fingerprint.py (hunk 1 above)
Comment (rilango) [reply common/cuchemcommon/fingerprint.py:13]: yes, that is correct.

We can move benchmark modules out into its own git repo if we find it useful.

Comment (mlgill) [reply common/cuchemcommon/fingerprint.py:13]: Thanks for clarifying. Agreed, we may need to move into it's own repo. Suggest we wait until needed though, if that's OK.

Comment (mlgill) [reply common/cuchemcommon/fingerprint.py:13]: Please feel free to merge when ready.

**Pull request merged**